File: src/main/java/sc02/Q01.java
package sc02;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Q01 {
    public static void main(String[] args) {
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
            String sentence = br.readLine().toLowerCase();
            char ch = br.readLine().toLowerCase().charAt(0);

            int count = 0;
            for (char c : sentence.toCharArray()) {
                if (c == ch) {
                    count++;
                }
            }

            System.out.println(count);
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }
}

File: src/main/kotlin/sc02/Q01.kt
package jp.co.kopher.sc02

import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader

fun main() {
    try {
        BufferedReader(InputStreamReader(System.`in`)).use { br ->
            val sentence = br.readLine().lowercase()
            val ch = br.readLine().lowercase()[0]

            var count = 0
            for (c in sentence.toCharArray()) {
                if (c == ch) {
                    count++
                }
            }
            println(count)
        }
    } catch (e: IOException) {
        throw RuntimeException(e)
    }
}